SpotlightDownloader/FileSystemAdmin.cs

```diff
@@ -1,16 +1,17 @@
 ﻿using System;
 using System.IO;
-using PrivilegeClass;
 using System.Security.AccessControl;
 using System.Security.Principal;
 
-namespace SharpTools
+using PrivilegeClass;
+
+namespace SpotlightDownloader
 {
     /// <summary>
     /// File system utilities for applications running as administrator
     /// By ORelio (c) 2018 - CDDL 1.0
     /// </summary>
-    public static class FileSystemAdmin
+    internal static class FileSystemAdmin
```

Comment:
This class being a companion for Privilege20, I think it can be safely removed after switching to the built-in wallpaper/lockscreen APIs.